projects/golang/go/1.18/rpmbuild/SPECS/golang.spec

```diff
@@ -164,6 +164,10 @@ Patch101:       0101-syscall-expose-IfInfomsg.X__ifi_pad-on-s390x.patch
 Patch102:       0102-cmd-go-disable-Google-s-proxy-and-sumdb.patch
 Patch103:       0103-Don-t-use-the-bundled-tzdata-at-runtime-except-for-t.patch
 Patch104:       0104-add-method-to-skip-privd-tests-if-required.patch
+Patch105:       0105-update-bundled-golang.org.patch
```

Comment:
Aren't the 100 series patches for our own patches to golang, and not the backported ones?